tools/reviewer/webapp/src/app/dashboard/file-changes/code-changes/services/template.service.ts

```diff
@@ -100,4 +100,16 @@ export class TemplateService {
   isEmpty(blockLine: BlockLine): boolean {
     return blockLine.threads.length === 0;
   }
+
+  isLastLine(changesLines: ChangesLine[], lineIndex: number): boolean {
```

Comment:
Can you please explain this method?
I.e why is it not simply `lineIndex === changesLines.length - 1;`

Reply:
:disappointed: round corners